src/EditorFeatures/Core/Implementation/Classification/IViewSupportsClassificationService.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
+{
+    internal interface IViewSupportsClassificationService
```

Comment:
Can you document what this interface is a bit, rather than it being another random floaty "magic bit to workaround a problem" interface with no documentation? (I say this as someone guilty of making those interfaces.)